lib/mirage.ml

```diff
@@ -1401,16 +1462,39 @@ let configure_makefile ~target ~root ~name ~warn_error info =
               export OPAMVERBOSE=1\n\
               export OPAMYES=1";
   newline fmt;
-  let pkg_config_deps = "mirage-xen" in
+  let pkg_config_deps =
+    match target with
+    | `Xen -> "mirage-xen"
+    | `Virtio -> "mirage-solo5 ocaml-freestanding"
+    | `Ukvm -> "mirage-solo5 ocaml-freestanding solo5-kernel-ukvm"
```

Comment:
not sure whether solo5-kernel-ukvm is really needed here, maybe @mato / @djwillia know details (used below with `pkg-config --static --libs %s`)?